renderer/components/config/index.js

```diff
@@ -0,0 +1,174 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {connect, ConfigContainer} from '../../containers';
+import Tab from './tab';
+
+class Config extends React.Component {
+  render() {
+    const {
+      validators,
+      values,
+      onChange,
+      selectedTab,
+      selectTab,
+      closeWindow,
+      openConfig,
+      viewOnGithub
+    } = this.props;
+
+    if (!validators) {
+      return null;
+    }
+
+    return (
+      <div className="container">
+        {
+          validators.length > 1 && (
+            <nav className="service-nav">
+              {
+                validators.map((validator, index) => {
+                  return (
+                    <div
+                      key={validator.title}
+                      className={selectedTab === index ? 'selected' : ''}
+                      onClick={() => selectTab(index)}
+                    >
+                      {validator.title}
+                    </div>
+                  );
+                })
+              }
+            </nav>
+          )
+        }
+        <div className="tab-container">
+          <div className="switcher"/>
+          {
+            validators.map(validator => {
+              return (
+                <div key={validator.title} className="tab">
+                  <Tab
+                    validator={validator}
+                    values={values}
+                    openConfig={openConfig}
+                    viewOnGithub={viewOnGithub}
+                    onChange={onChange}
+                  />
+                </div>
+              );
+            })
+          }
+        </div>
+        <footer>
+          <div className="fade"/>
+          <div className="button" onClick={closeWindow}>Done</div>
+        </footer>
+        <style jsx>{`
+          .container {
+            height: 100%;
+            width: 100%;
+            display: flex;
+            flex-direction: column;
+          }
+
+          .service-nav {
+            height: 3.6rem;
+            padding: 0 16px;
+            display: flex;
+            align-items: center;
+            box-shadow: 0 1px 0 0 #ddd, inset 0 1px 0 0 #fff;
+            z-index: 10;
+            max-width: 100%;
+            overflow-x: auto;
+          }
+
+          .service-nav div {
+            margin-right: 16px;
+            height: 100%;
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            padding-bottom: 2px;
+            font-size: 1.2rem;
+            color: #007AFF;
```

Comment:
We should really make use of CSS variables. I see this color used constantly. Using CSS variables will be important for adding dark mode support too. // @skllcrn

Reply:
I'll open a new PR that takes out the colors into variables 👍